src/components/HomeMain.component/HomeMain.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import HomeCategories from "./Categories.component/HomeCategories";
+import Hero from "./Hero.component/Hero";
+import HomeTrendingCard from "./HomeTrendingCard.component/HomeTrendingCard";
+import "./home-main.css";
+const HomeMain = () => {
+  return (
+    <main className="home-main">
+      <section className="categories">
+        <HomeCategories />
+        <HomeCategories />
+        <HomeCategories />
+        <HomeCategories />
+        <HomeCategories />
```

Comment:
You can map your HomeCategories instead of importing components so many time.

Reply:
Data is not implemented yet. This is just the screen.